src/main/resources/init_db.sql

```diff
@@ -0,0 +1,12 @@
+CREATE SCHEMA `taxi_service` DEFAULT CHARACTER SET utf8;
+
+CREATE TABLE `manufacturers`
+(
+    `manufacturer_id`      bigint       NOT NULL AUTO_INCREMENT,
+    `manufacturer_name`    varchar(100) NOT NULL,
+    `manufacturer_country` varchar(100) NOT NULL,
+    `deleted`              tinyint(1)   NOT NULL DEFAULT '0',
+    PRIMARY KEY (`manufacturer_id`)
+) ENGINE = InnoDB
+  AUTO_INCREMENT = 7
```

Comment:
why 7 ?

Reply:
I wish to know, it is a copy of `CREATE` script from MySQL workbench.